apps/react-workshop/src/LabeledTextarea.test.ts

```diff
@@ -21,4 +21,19 @@ describe('LabeledTextarea', () => {
       cy.compareSnapshot(testName);
     });
   });
+
+  // To also test that the correct status outline is applied
+  ['Positive', 'Warning', 'Negative'].forEach((testName) => {
```

Comment:
There are tests in `css-workshop` for this. Adding this in `react-workshop` too since the [status outline not applied issue](https://github.com/iTwin/iTwinUI/pull/1753#discussion_r1437277459) was only on the react side.